config/default.yml

```diff
@@ -82,6 +82,8 @@ AllCops:
   StyleGuideBaseURL: https://rubystyle.guide
   # Documentation URLs will be constructed using the base URL.
   DocumentationBaseURL: https://docs.rubocop.org/rubocop
+  # Documentation URLs will end with this extension
+  DocumentationFormat: .html
```

Comment:
Won't it be better to specify the actual format here and just infer the extension from it?

Reply:
You mean something like `application/html`? I don't think I'm following.

Reply:
I thought about "markdown" instead of "md", but this was before I actual saw what needed to be changed. :-)

Reply:
Because the name is `DocumentationFormat` rather than `DocumentationExtension`, `markdown` felt like a more appropriate value than `.md`. It would be better to match the name and the value.

Reply:
This makes sense, thanks. I renamed the option.